microfit/run_plotter.py

```diff
@@ -144,6 +144,7 @@ def flatten(hist):
             ax.set_xlabel(total_pred_hist.binning.label)
             return ax
 
+        # TODO: Xheck this
```

Comment:
There will probably not be one universal ratio that is good for every variable. We can try to find something smarter in the future.